src/components/scs/ScSheet/ScDelegationMenu.js

```diff
@@ -0,0 +1,176 @@
+import React, { Fragment, useState } from 'react';
+import { injectIntl } from 'react-intl';
+import { withStyles } from '@material-ui/core';
+import Typography from '@material-ui/core/Typography';
+import EmployeeFilter from '../../admin/EmployeeFilter';
+import Button from '@material-ui/core/Button';
+import getDisplayName from '../../../helper/getDisplayName';
+import { saveDelegation } from '../../../calls/sc';
+
+const styles = theme => ({
+  spacing: {
+    margin: 3 * theme.spacing.unit
+  },
+  saveBtn: {
+    margin: 2 * theme.spacing.unit
+  },
+  root: {
+    padding: 3 * theme.spacing.unit
+  },
+  avatar: {
+    backgroundColor: theme.palette.primary[500]
+  },
+  heading: {
+    flex: '1 1 auto'
+  },
+  names: {
+    display: 'flex',
+    '& p:not(:first-child)': {
+      paddingLeft: theme.spacing.unit
+    }
+  }
+});
+
+const ScDelegationMenu = ({
+  classes,
+  sc,
+  intl,
+  employees,
+  info,
+  error,
+  activeDelegationButtons
+}) => {
+  const [scReviewers, setScReviewers] = useState({
+    reviewer1: sc.reviewer1,
+    reviewer2: sc.reviewer2
+  });
+  const [selectedReviewer1, setSelectedReviewer1] = useState(sc.reviewer1);
+  const [selectedReviewer2, setSelectedReviewer2] = useState(sc.reviewer2);
+
+  const addReviewer = employee => {
+    if (selectedReviewer1) {
+      setSelectedReviewer2(employee);
+    } else {
+      setSelectedReviewer1(employee);
+    }
+  };
+
+  const im = id =>
+    intl.formatMessage({
+      id: id
+    });
+
+  const supervisorAndReviewersText = () => {
+    //Supervisor: Michal Beres,
+    let resultText = `${im('sc.supervisor')} ${getDisplayName(
+      sc.supervisor
+    )}, `;
+
+    //Supervisor: Michael Beres, Kein Reviewer
```

Comment:
Please remove redundant comments

Reply:
It was the intention - to have a an overview, what type of message I m handling in the code... otherwise it s harder to understand